ontologies/semopenalex-ontology.ttl

```diff
@@ -0,0 +1,1136 @@
+  sh:property <https://semopenalex.org/vocab#23348fdc99b5798e624fe091e0b4709d>, <https://semopenalex.org/vocab#330235a73dd4a216315cf40345bf056b>,
+    <https://semopenalex.org/vocab#3494e2ce7122eeac1b765ba1aa7e89f7>, <https://semopenalex.org/vocab#f560b07f741a900b03806de5c0ba9e2c>,
+    [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/hasSource>;
+      sh:class <https://semopenalex.org/class/Source>
+    ], [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/pdfUrl>;
+      sh:datatype xsd:string
+    ];
+  sh:targetClass <https://semopenalex.org/class/Location> .
+
+<https://semopenalex.org/vocab#23348fdc99b5798e624fe091e0b4709d> a sh:PropertyShape;
+  sh:path <http://purl.org/spar/fabio/hasURL>;
+  sh:datatype xsd:anyURI .
+
+<https://semopenalex.org/vocab#330235a73dd4a216315cf40345bf056b> a sh:PropertyShape;
+  sh:path <https://semopenalex.org/property/hasVersion>;
+  sh:datatype xsd:string .
+
+<https://semopenalex.org/vocab#3494e2ce7122eeac1b765ba1aa7e89f7> a sh:PropertyShape;
+  sh:path <http://purl.org/dc/terms/license>;
+  sh:datatype xsd:string .
+
+<https://semopenalex.org/vocab#f560b07f741a900b03806de5c0ba9e2c> a sh:PropertyShape;
+  sh:path <https://semopenalex.org/property/isOa>;
+  sh:or ([
+        sh:class <https://semopenalex.org/class/Source>
+      ] [
+        sh:class <https://semopenalex.org/class/Location>
+      ] [
+        sh:class <https://semopenalex.org/class/OpenAccess>
+      ]);
+  sh:datatype xsd:boolean .
+
+<https://semopenalex.org/class/InstitutionShape> a sh:NodeShape;
+  sh:property <https://semopenalex.org/vocab#7e72e214b08c0478192cb924f8fdd396>, <https://semopenalex.org/vocab#3e2f60daa912b4bbfdda79d75fe2752b>,
+    <https://semopenalex.org/vocab#534f665d25d906da0a8c33c199207f86>, <https://semopenalex.org/vocab#893797ba8898177df5b817ba86554a38>,
+    <https://semopenalex.org/vocab#b24dbbf3333905048d57498219f7c8c2>, <https://semopenalex.org/vocab#bb72667cc1ad07ac0375082b6b947fe1>,
+    <https://semopenalex.org/vocab#f31288f45557f654fc2db4a369bebfa7>, <https://semopenalex.org/vocab#07fe8d090dee52aa25cacf9ba01c537d>,
+    <https://semopenalex.org/vocab#d780b88e0412f91d5725d74b7b2da814>, <https://semopenalex.org/vocab#0d30180e88607a9b9f3e7efdf6a5a920>,
+    <https://semopenalex.org/vocab#32acd9478deb48037209d21c271560b9>, <https://semopenalex.org/vocab#4664cd9b195ba95e5b19f26ac9baf427>,
+    <https://semopenalex.org/vocab#4832ab1df642e9e99ed61f38e2431b26>, <https://semopenalex.org/vocab#6ea9822eaeb42efcce076ee379679293>,
+    <https://semopenalex.org/vocab#99e3c941a405656773ad39ea7e98232e>, <https://semopenalex.org/vocab#cd3685d05f804b6eb5d2ec436d6302d4>,
+    <https://semopenalex.org/vocab#e6d6f1fbf0cd5a66ac9208072882dea6>, <https://semopenalex.org/vocab#fe3d9ddc90fd4837401ab65fb4f1b962>;
+  sh:targetClass <https://semopenalex.org/class/Institution> .
+
+<https://semopenalex.org/vocab#0d30180e88607a9b9f3e7efdf6a5a920> a sh:PropertyShape;
```

Comment:
to my understanding this shape (and the following one is invalid)

it cannot have sh:class (relation) and sh:datatype (attribute) at the same time ?